dev/KY-008.go

```diff
@@ -0,0 +1,52 @@
+/*Package dev ...
+
+Connect to Pi:
+- positive: 	    	any data pin
+- negative(marked): 	any gnd pin
+*/
+
+package dev
+
+import (
+	"github.com/stianeikeland/go-rpio"
+	"time"
+)
+
+const (
+	logTagLaser = "laser"
+)
+
+// Laser ...
+type Laser struct {
```

Comment:
it would better to name it as `type KY008 struct`, let its name the same as the filename.
assume there is another laser module KY-009 we will not know how to name it if `laser` is occupied.